commands/dev-deploy.js

```diff
@@ -63,7 +64,8 @@ async function createDevAccountIfNeeded({ near, keyStore, networkId, init }) {
             }
         } catch (e) {
             if (e.code === 'ENOENT') {
-                // Ignore as it means new account needs to be created, which happens below
+                // Create neardev directory, new account will be created below
+                if (!existsSync('./neardev')) await mkdir('./neardev');
```

Comment:
let's pull the dir name into const, so that we don't have string literal duplication

Reply:
pushed